### DISCUSSION TITLE
Cannot target tailwind classes

### DISCUSSION THREAD

**Discussion by meshakeeb** [tailwindlabs/tailwindcss#3733] [Help]
### Describe the problem:

I have a class with which I want to target the children with `w-full` class so I did something like this. I am using laravel-mix as my workflow manager

```
.logos {
	.w-full > div {
		@apply inline-flex p-3 items-center justify-center mb-6 h-20;
	}
}
```

**Comment by aggreggator** [comment]
Which selector do you expect to get?

**Comment by meshakeeb** [comment]
> Which selector do you expect to get?

Sorry I edit the question for better understanding as soon as I removed `.w-full > div` or renamed it the css got compiled.

**Comment by bradlc** [comment]
Hey @meshakeeb. What is the issue that you're having exactly? Are you seeing some sort of error? The first thing to check would be that you have a nesting plugin installed, such as [PostCSS Nested](https://github.com/postcss/postcss-nested).

**Comment by meshakeeb** [comment]
Hi @bradlc ,

I am using `node-sass` in my workflow the issue is I cant target elements with utility classes the above code is compiling and no error thrown.

Thanks

**Comment by bradlc** [comment]
Ah so if I understand correctly, your markup is something like this:

```html
<div class="logos">
  <!-- you want to target the child divs that have the w-full class? -->
  <div class="w-full"></div>
  <div class="w-full"></div>
  <div></div>
</div>
```

If that's correct your CSS should be:

```css
.logos > .w-full {
  @apply inline-flex p-3 items-center justify-center mb-6 h-20;
}
```

Or if you prefer to nest:

```css
.logos {
  > .w-full {
    @apply inline-flex p-3 items-center justify-center mb-6 h-20;
  }
}
```

**Comment by meshakeeb** [comment]
I prefer to nest as I am using `SCSS` and this css is udner `@layer components`

**Comment by bradlc** [comment]
Did I understand your issue correctly, and did the CSS I provided solve it?

**Comment by meshakeeb** [comment]
Hi this is the correct CSS which I was trying to compile but its not compiling and there was no error on cli as well.

```
.logos {
  > .w-full {
    @apply inline-flex p-3 items-center justify-center mb-6 h-20;
  }
}
```

But as soon as I rename `w-full` to `w-full1` it gets compiled so the below css is compiling fine.

```
.logos {
  > .w-full1 {
    @apply inline-flex p-3 items-center justify-center mb-6 h-20;
  }
}
```

**Comment by bradlc** [comment]
I tried to reproduce this but I wasn't able to: https://github.com/bradlc/laravel-mix-tailwind-sass

Running `npm run build` in this project results in the expected CSS (`dist/app.css`):

```css
.logos > .w-full {
  display: inline-flex;
  align-items: center;
  justify-content: center;
  height: 5rem;
  margin-bottom: 1.5rem;
  padding: 0.75rem;
}
```

Please can you share a repo which reproduces the issue? Otherwise it will be basically impossible to debug, thanks!

**Comment by meshakeeb** [comment]
Hi @bradlc appreciate your efforts here is the mix file I am using.

```
/**
 * External Dependencies
 */
const mix = require( 'laravel-mix' )
const classNames = require( 'classnames' )

/**
 * Internal Dependencies
 */
require( './laravel-mix/nunjucks' )
require( 'laravel-mix-tailwind' )

mix.webpackConfig(
	{
		externals: {
			jquery: 'jQuery'
		}
	}
)

/**
 * HTML
 */
mix.njk(
	'src/html/*.html',
	'dist',
	{
		ext: '.html',
		data: {
			author: 'Shakeeb Ahmed',
			templateName: 'Sobrado',
			classNames: classNames,
		},
		partials: [
			'src/html/layouts/**/*.html',
			'src/html/macros/**/*.html',
			'src/html/partials/**/*.html',
		]
	}
)

/**
 * CSS
 */
mix.sass(
	'src/scss/theme.scss',
	'dist/assets/css/theme.css',
).tailwind()

/**
 * JavaScript
 */
mix.js(
	'src/js/index.js',
	'dist/assets/js/app.js'
).sourceMaps( true, 'source-map' )

mix.copy( 'src/vendors', 'dist/assets/vendors');

/**
 * Images
 */
mix.copy( 'src/img', 'dist/assets/img');

/**
 * Fonts
 */
mix.copy( 'src/fonts', 'dist/assets/fonts');

/**
 * Browsersync
 */
mix.browserSync( {
	server: {
		baseDir: 'dist'
	},
	files: [
		'dist/*.html',
		'dist/assets/css/*.css',
		'dist/assets/js/*.js',
	],
} )
```

**Comment by meshakeeb** [comment]
Any luck so far
